### PR TITLE
feat[docs]: add Telegram badge to README.md

### PR DESCRIPTION
**Pull request by rafael-abuawad**
### What I did
Added Telegram badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Build Status](https://github.com/vyperlang/vyper/workflows/Test/badge.svg)](https://github.com/vyperlang/vyper/actions/workflows/test.yml)
 [![Documentation Status](https://readthedocs.org/projects/vyper/badge/?version=latest)](http://docs.vyperlang.org/en/latest/?badge=latest "ReadTheDocs")
 [![Discord](https://img.shields.io/discord/969926564286459934.svg?label=%23vyper)](https://discord.gg/6tw7PTM7C2)
+[![Telegram](https://img.shields.io/badge/Vypeholics🐍-Telegram-blue)](https://t.me/vyperlang)
```
Comment (charles-cooper) [inline README.md:6]:
```suggestion
[![Telegram](https://img.shields.io/badge/Vyperholics🐍-Telegram-blue)](https://t.me/vyperlang)
```

**Comment by rafael-abuawad** [comment]
@charles-cooper can we get this merged?

**Review by charles-cooper — APPROVED** [review]

**Comment by charles-cooper** [comment]
looks like the windows build doesn't like the existence of that emoji

**Comment by charles-cooper** [comment]
this seems to be a pyinstaller issue, which i've raised upstream https://github.com/pyinstaller/pyinstaller/issues/8865

**Comment by charles-cooper** [comment]
i have a fix in https://github.com/vyperlang/vyper/pull/4348/files

**Pull request merged**